prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -329,7 +330,9 @@ class FhirPathCustomResolver : FHIRPathEngine.IEvaluationContext, Logging {
         return if (constantValue.isNullOrBlank()) null
         else {
             val values = FhirPathUtils.evaluate(appContext, appContext.focusResource, appContext.bundle, constantValue)
-            if (values.size != 1) {
+            if (values.isEmpty()) {
+                null
+            } else if (values.size != 1) {
```

Comment:
Concerned at all that if your mistype your constant, or it is multi valued the cli tool just boots you out?

Reply:
Added an extra check, so now it will not blow up